### DISCUSSION TITLE
Should I use maftools to create like this plot?

### DISCUSSION THREAD

**Discussion by kwanghoon821** [PoisonAlien/maftools#659] [Q&A]
https://www.researchgate.net/publication/227857324_Novel_mutations_target_distinct_subgroups_of_medulloblastoma/figures?lo=1

I'm trying to draw plot like above (figure1).

It looks like that plot is similar to maftools' output.

Is it possible to draw that plot with excel file?

**Comment by PoisonAlien** [comment]
Hi,
It depends on the file format. If the contents of your excel file matches the [MAF](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) format, then you can just export it to TSV file and process it with maftools.
